packages/tutorial/src/part4.tsx

```diff
@@ -0,0 +1,38 @@
+import fetch from 'node-fetch';
+import { htmlToText } from 'html-to-text';
+import * as LLMx from 'ai-jsx';
+import { ChatCompletion, SystemMessage, UserMessage } from 'ai-jsx/core/completion';
+import { DocsQA, ScoredChunk, LocalCorpus, makeChunker, staticLoader } from 'ai-jsx/batteries/docs';
+import { memo } from 'ai-jsx/core/memoize';
+import { showInspector } from 'ai-jsx/core/inspector';
+
+const URL = 'https://en.wikipedia.org/wiki/2005_Azores_subtropical_storm';
+
+const html = await fetch(URL).then((response) => response.text());
+const plainText = htmlToText(html);
+const docs = [
+  {
+    pageContent: [plainText],
+    name: 'Wikipedia Article about 2005 Azores subtropical storm',
+  },
+];
+const corpus = new LocalCorpus(staticLoader(docs), makeChunker(300, 100));
```

Comment:
any reason you changed the chunk size here?

Reply:
Couldn't get it to work with the larger size -- was getting errors from the model that we'd exceeded the token limit.

Reply:
With Justin's change, I suspect you could reduce the limit on chunks returned by the corpus instead of reducing chunk size.  Up to you.

Reply:
Yeah, I think it would be better to reduce the # of chunks, 300 is starting to get on the small side for a chunk.

Reply:
How do I do this? The only two parameters I see are the chunk size and overlap. You mean 'limit' ?

Reply:
https://github.com/fixie-ai/ai-jsx/blob/e9f20387c905d42b3c8f84b902502a116f4d5474/packages/create-react-app-demo/src/docs-chat/ai.tsx#L44